src/components/layout/sendTip/SendTip.vue

```diff
@@ -111,12 +111,12 @@ export default {
         .then(async () => {
           await Backend.cacheInvalidateTips().catch(console.error);
           this.clearTipForm();
-          EventBus.$emit('reloadData');
           this.$store.dispatch('modals/open', {
             name: 'success',
             title: this.$t('components.layout.SendTip.SuccessHeader'),
             body: this.$t('components.layout.SendTip.SuccessText'),
           });
+          setTimeout(() => EventBus.$emit('reloadData'), 5000);
```

Comment:
Why is this timeout needed here?

Reply:
The backend itself take about 5 secs to process the tip, so it was suggested not to reload immediately after sending the tip, but delay a little bit.

Reply:
Is delay of `5000` ms enough?

Reply:
with 5000 ms delay for sending the reload request plus 12 000 ms timeout before throwing the error even for slow connections should be fine.